python/pyarrow/_csv.pyx

```diff
@@ -58,6 +58,7 @@ cdef class ReadOptions(_Weakrefable):
         How much bytes to process at a time from the input stream.
         This will determine multi-threading granularity as well as
         the size of individual record batches or table chunks.
+        Minimum valid value for block size is 1KB
```

Comment:
You mean 1B, no?

Reply:
Ooops, I wanted 1KB but tripped over tests. Guess I missed a spot